com.musala.xml.parser.executable/hs_err_pid4488.log

```diff
@@ -0,0 +1,151 @@
+#
```

Comment:
All files of that type are error logs for memory leaks, usually caused by insufficient memory. They are generated by the JVM for informative purposes, so please delete all of them.